Sources/AppcuesKit/Presentation/Actions/Appcues/AppcuesTrackAction.swift

```diff
@@ -10,27 +10,68 @@ import Foundation
 
 @available(iOS 13.0, *)
 internal class AppcuesTrackAction: AppcuesExperienceAction {
-    struct Config: Decodable {
+    struct Config {
         let eventName: String
+        let attributes: [String: Any]?
+    }
+    
+    private enum CodingKeys: String, CodingKey {
+           case eventName
+           case attributes
     }
```

Comment:
Minor organization nitpick: I'd move this down inside the extension on L48.

Then you can reference `CodingKeys.self` instead of `AppcuesTrackAction.CodingKeys.self`